cinema/views.py

```diff
@@ -40,6 +60,124 @@ def movie_detail(request, pk):
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    if request.method == "PATCH":
+        serializer = MovieSerializer(movie, data=request.data, partial=True)
+        if serializer.is_valid():
```

Comment:
use serializer.is_valid(raise_exception=True)
and remove the if statement. Fix in all places